test/evergreen.yml

```diff
@@ -1414,7 +1424,7 @@ variables:
         export LD_LIBRARY_PATH=""
         export DYLD_LIBRARY_PATH=$WT_BUILDDIR
       # Must disable TCMALLOC as it may be picked up locally and its not on all hosts.
-      ENABLE_TCMALLOC: -DENABLE_TCMALLOC=0
+      ENABLE_TCMALLOC: ENABLE_TCMALLOC=0
```

Comment:
I'm a little confused about the environment variable business. So is `ENABLE_TCMALLOC=0` still passed into CMake? If so - how does it use it without the `-D` prefix?

And if not - should we take the opportunity to decrease the verbosity of this and have `ENABLE_TCMALLOC: 0`?

Reply:
I have cleaned this up to be an environment only numeric boolean as discussed.